packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -146,6 +146,8 @@ export const CheckboxInput = React.forwardRef<HTMLInputElement, InputProps>(
   },
 );
 
+CheckboxInput.displayName = "CheckboxInput";
```

Comment:
There's now a React lint rule enforcing providing a `displayName` where it doesn't think one will be inferred (such as when using `forwardRef`).